### DISCUSSION TITLE
GL-MT300N-V2 support

### DISCUSSION THREAD

**Discussion by Andromeda1966** [Ysurac/openmptcprouter#1852] [Q&A]
Hi,
is there a way to install openmptcprouter on a GL-MT300N-V2 ?
Thanks for any information.

**Comment by Ysurac** [comment]
You can compile it yourself using https://github.com/Ysurac/openmptcprouter/wiki/Create-image-for-unsupported-platform
But this device have a slow CPU, small flash and small RAM with LAN port limited to 100Mbs, the only good things is the AES support that I think is available.
